config/nunjucks.config.js

```diff
@@ -69,7 +69,7 @@ function configure(env) {
   });
 
   env.addFilter('mbtogb', (mb) => {
-    return `${(mb / 1024).toFixed(2)}<abbr title="gigabytes">gb</abbr>`;
+    return `${(mb / 1024).toFixed(2)}<abbr title="gigabytes">GB</abbr>`;
```

Comment:
This should be GiB technically - it's a Gibibyte - https://en.wikipedia.org/wiki/Gibibyte

Reply:
❇️